parts/main.fodt

```diff
@@ -264148,7 +264148,7 @@ Updated with AFR/TSA Rev-D comments and new  keywords.</dc:description><meta:ini
        <text:p text:style-name="Table_20_Contents"><text:s text:c="2"></text:s>Document Name</text:p>
       </table:table-cell>
       <table:table-cell table:style-name="Table1883.B22" table:number-columns-spanned="4" office:value-type="string">
-       <text:p text:style-name="P10483"><text:file-name text:display="full">davs://blattms@storage.opm-op.com/remote.php/dav/files/blattms/opm-op/Projects/P20-07 Equinor Maintenance/Reference Manual/2023-04 Manual/OPM_Flow_Reference_Manual_2023-04_Rev-0.odt</text:file-name></text:p>
+       <text:p text:style-name="P10483"><text:file-name text:display="full">OPM_Flow_Reference_Manual_2023-0am_Rev-A.fdot</text:file-name></text:p>
```

Comment:
This is what I see when I open main.fodt:

![image](https://github.com/OPM/opm-reference-manual/assets/6708379/2c731152-7ef9-4be1-a8e1-5d3a7585fbb8)

Where do the text `/home/docker-user/parts/main.fodt` come from in the screenshot above?

Reply:
This might always be substituted by the current location. I don't know why. maybe a macro does this? We could just delete this row. It has now real value.

Reply:
Maybe we should also do something about the email. Matthew might receive a lot spam because of this.

Reply:
> Where does the text `/home/docker-user/parts/main.fodt` come from in the screenshot above?

@blattms I think it could be due to the `<text:file-name>` tag? If select the item and then right-click on it, you can choose "Edit Fields" from the popup menu:

![image](https://github.com/OPM/opm-reference-manual/assets/6708379/1bac4ab5-7316-48f1-8c9a-83b51fd3bdd6)

Which gives this dialog:

![image](https://github.com/OPM/opm-reference-manual/assets/6708379/26bbaf17-4ea9-42f2-ac02-aac34b2c17f2)

Reply:
> Maybe we should also do something about the email. Matthew might receive a lot spam because of this.

@blattms  I hope not. @gdfldm Let me know if you want to obfuscate the email, or I can merge this now.

Reply:
@blattms perhaps it would be better to suggest people raise issues?

Reply:
I removed the file-name tag. Thanks a lot for pointing that out @hakonhagland 
I now point to github instead of listing an email,